vmdb/spec/factories/provider.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :foreman_provider, :class => 'ProviderForeman' do
```

Comment:
I just changed it to cleanup the rubocop comment.  I feel like all providers will implement authentications in their own way.  Once we have a couple more then we can refactor?